File: utbot-spring-analyzer/src/main/java/application/TestApplicationConfiguration.java
package application;

import org.springframework.beans.factory.config.BeanFactoryPostProcessor;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import post_processors.UtBotBeanFactoryPostProcessor;

@Configuration
public class TestApplicationConfiguration {

    @Bean
    public static BeanFactoryPostProcessor utBotBeanFactoryPostProcessor() {
        return new UtBotBeanFactoryPostProcessor();
    }
}

File: utbot-spring-analyzer/src/main/kotlin/org/utbot/spring/config/TestApplicationConfiguration.kt
package org.utbot.spring.config

import org.springframework.beans.factory.config.BeanFactoryPostProcessor
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.ImportResource
import org.utbot.spring.postProcessors.UtBotBeanFactoryPostProcessor

@Configuration
@ImportResource
open class TestApplicationConfiguration {

    @Bean
    open fun utBotBeanFactoryPostProcessor(): BeanFactoryPostProcessor = UtBotBeanFactoryPostProcessor
}